TAF/testScenarios/integrationTest/UC_device_service/device_configuration.robot

```diff
@@ -2,6 +2,7 @@
 Documentation  Configrations
 Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
 Resource     TAF/testCaseModules/keywords/device-sdk/deviceServiceAPI.robot
+Resource  TAF/testCaseModules/keywords/core-keeper/coreKeeperAPI.robot
```

Comment:
This resource shouldn't be imported if using keyword `Update Service Configuration` to update configuration.